dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -297,7 +297,7 @@ def redirect_facilitator(key_params)
       if next_facilitator_index.between?(1, session.workshop.facilitators.size - 1)
         redirect_to action: :new_facilitator, session_id: session.id, facilitator_index: next_facilitator_index
       # No facilitators left. Academic workshops redirect to post if its the last day
-      elsif !session.workshop.summer? && key_params[:day].to_i == session.workshop.sessions.size
+      elsif !session.workshop.summer? && !session.workshop.csf_201? && key_params[:day].to_i == session.workshop.sessions.size
```

Comment:
Can we add comments explaining?

Reply:
Also is there a way to generalise this behaviour so that future workshops with the same need can somehow declare they get the same treatment, rather than hardcoding the behaviour here in the controller?

Reply:
I will add comment. The special logic applies only to Academic year workshops, which we have so many different subjects, so it's actually easier to write a _not academic-year workshop_ condition. 
Let's revisit configuring workshop for survey behavior later; I think it's a bigger discussion.